app/src/main/java/com/github/se/eduverse/MainActivity.kt

```diff
@@ -75,12 +75,12 @@ class MainActivity : ComponentActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
 
     super.onCreate(savedInstanceState)
-
+    /*
     // Initialiser Firebase Auth
     auth = FirebaseAuth.getInstance()
     if (auth.currentUser != null) {
       auth.signOut()
-    }
+    } */
 
```

Comment:
This was also causing the bug exposed in #114, and was resolve by deleting this part of the code in #120 (not merged at this time). This will cause a conflict in whichever is merged last, but it is fine as both version are equivalent.

Reply:
As there was no point in keeping both, I deleted said PR, so we're all good👍